hvplot/tests/testpanel.py

```diff
@@ -59,11 +59,10 @@ def test_casting_widgets_to_different_classes(self):
 
         assert len(look_for_class(pane, pn.widgets.DiscreteSlider)) == 1
 
-    def test_using_explicit_widgets_with_groupby_raises_error(self):
+    def test_using_explicit_widgets_with_groupby_does_not_raise_error(self):
         import panel as pn
 
         x = pn.widgets.Select(name='x', value='sepal_length', options=self.cols)
         y = pn.widgets.Select(name='y', value='sepal_width', options=self.cols)
 
-        with self.assertRaisesRegex(ValueError, "Groupby is not yet"):
-            self.flowers.hvplot(x, y, groupby='species')
+        self.flowers.hvplot(x, y, groupby='species')
```

Comment:
Could the test actually check the structure of the output?

Reply:
I really couldn't figure out how to check that.

Reply:
Can't unpack the different components and perform isinstance checks?